### ISSUE TITLE
Evaluation on quantized SSD mobilenetV1

### ISSUE THREAD

**Issue by mmanoharahere** [tensorflow/models#7584]
Hello,

I have added the following to pipeline.config:
graph_rewriter {
  quantization {
    delay: 0
    activation_bits: 8
    weight_bits: 8
  }
}

During quantization-aware training, i can see that the trend of loss decrease is similar to fine-tuning checkpoints.
When I evaluate the quantized checkpoints, I get an mAP of 1% vs fine-tuned checkpoints mAPof 23%.

Is there anything else that I need to add to the config file to evaluate quantized checkpoints correctly?

Any help is appreciated.
Thank you!

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
